cleverhans/attacks.py

```diff
@@ -749,6 +749,79 @@ def parse_params(self, y=None, y_target=None, nb_classes=None,
         self.clip_max = clip_max
 
 
+class DeepFool(Attack):
+
+    """
+    DeepFool paper link: "https://arxiv.org/pdf/1511.04599.pdf"
```

Comment:
Could you add a short description of the attack to explain what the attack is doing in a few words?